cfme/tests/automate/test_rest.py

```diff
@@ -54,11 +60,97 @@ def test_delete_automate_domain_from_detail(appliance, domain_rest, method):
 
 
 @pytest.mark.ignore_stream("5.10")
-def test_delete_automate_domain_from_collection(appliance, domain_rest):
+def test_delete_automate_domain_from_collection(domain_rest):
     """
     Polarion:
         assignee: pvala
         casecomponent: Automate
         initialEstimate: 1/10h
     """
     delete_resources_from_collection([domain_rest], not_found=True, num_sec=50)
+
+
+@pytest.mark.ignore_stream("5.10")
+@pytest.mark.tier(2)
+@pytest.mark.meta(
+    automates=[1486765, 1740340],
+    blockers=[BZ(1740340, unblock=lambda scheduler: scheduler != "exact_time")],
+)
+@pytest.mark.parametrize("scheduler", ["number_of_days", "exact_time"])
+@test_requirements.rest
+def test_schedule_automation_request(appliance, scheduler):
+    """
+    Bugzilla:
+        1740340
+        1486765
+
+    Polarion:
+        assignee: pvala
+        caseimportance: high
+        casecomponent: Rest
+        initialEstimate: 1/4h
+        testSteps:
+            1. Send a request POST /api/automation_requests
+                {
+                    "uri_parts" : {
+                        "namespace" : "System",
+                        "class"     : "Request",
+                        "instance"  : "InspectME",
+                        "message"   : "create"
+                    },
+                    "parameters" : {
+                        "var1" : "value 1",
+                        "var2" : "value 2",
+                        "minimum_memory" : 2048,
+                        "schedule_time": scheduler
+                    },
+                    "requester" : {
+                        "auto_approve" : true
+                    }
+                }
+            2. Compare the `created_on` and `options::schedule_time` from the response.
+        expectedResults:
+            1. Request must be successful.
+            2.Difference between the two dates must be equal to scheduler
+    """
+    schedule_time = "2" if scheduler == "number_of_days" else "2019-08-14 17:41:06 UTC"
+    automate_request_rest = appliance.rest_api.collections.automation_requests.action.create(
+        {
+            "uri_parts": {
+                "namespace": "System",
+                "class": "Request",
+                "instance": "InspectME",
+                "message": "create",
+            },
+            "parameters": {
+                "var1": "value 1",
+                "var2": "value 2",
+                "minimum_memory": 2048,
+                "schedule_time": schedule_time,
+            },
+            "requester": {"auto_approve": True},
+        }
+    )[0]
+    assert_response(appliance)
+
+    automate_request = appliance.collections.automation_requests.instantiate(
+        description=automate_request_rest.description
+    )
+
+    # This step tests another error that occurred when navigating to the Details page
+    view = navigate_to(automate_request, "Details")
+    assert view.is_displayed
+
+    def _convert(date):
```

Comment:
Bit of overkill putting this in a function, could just define date_format as 'constant' and use the same datetime.strptime call using it.